### ISSUE TITLE
insert and update error with PostgreSQL Wire Protocol

### ISSUE THREAD

**Issue by nicetip** [crate/crate#7313]
**CrateDB version**: 3.0.0

**Environment description**:

**Problem description**:

In crate 3.0.0, following code occurs error.
It worked well in 2.3.x versions.
Look at last line.

```python
import psycopg2
conn = psycopg2.connect("dbname='mydb' host='127.0.0.1' port=5432")
cs = conn.cursor()
cs.execute("create table if not exists doc.test (col string)")
cs.execute("insert into doc.test (col) values (%s)", ("aaa",))
cs.execute("insert into doc.test (col) values (%s)", ("aaa'",))  # single-quote only. OK.
cs.execute("insert into doc.test (col) values (%s)", ("aaa;",))  # semi-colon only. OK.
cs.execute("insert into doc.test (col) values (%s)", ("aaa';",))  # single-quote and semi-colon. error.
```

I found this, by embedding some javascript code in column, inserting like this:
```javascript
var x = 'text';
```

Thanks.

**Steps to reproduce**:

**Comment by mxm** [comment]
Thank you for reporting. There is a fix pending for this in https://github.com/crate/crate/pull/7323. Will be included in 3.0.1.

**Comment by mxm** [comment]
Will be included in the next hotfix release.
